01/app.js

```diff
@@ -1 +1,9 @@
-console.log('DOM');
\ No newline at end of file
+console.log('DOM');
+
+const el1 = document.querySelector(".comments__item" && ".comments__item--newest")
```

Comment:
Prawidłowy zapis to `".comments__item.comments__item--newest"` - to co ty napisałeś daje `".comments__item--newest"` dlatego "działa", ale nie spełnia założeń zadania